h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -371,12 +371,20 @@ public static Vec makeCon(double x, long len, int log_rows_per_chunk) {
     return makeCon(x,len,log_rows_per_chunk,true);
   }
 
+  private static int VALUE_MAX = 1<<28; //original = 1<<28 = Value.MAX / 8
   /**
    * Make a new constant vector with minimal number of chunks. Used for importing SQL tables.
    *  @return New constant vector with the given row count. */
   public static Vec makeCon(long totSize, long len) {
-    int safetyInflationFactor = 8;
-    int nchunks = (int) Math.max(safetyInflationFactor * totSize / Value.MAX , 1);
+      return makeCon(totSize, len, VALUE_MAX);
+  }
+
+  public static Vec makeCon(long totSize, long len, int valueMax) {
```

Comment:
What is this method for?

Reply:
not used anymore, was used during benchmarks when changing the default magic valueMax.
will remove